app/controllers/ConfirmationController.scala

```diff
@@ -41,17 +42,28 @@ class ConfirmationController @Inject()(
                                         dataCacheConnector: DataCacheConnector,
                                         countriesService: CountriesService,
                                         pdfService: PdfService,
+                                        btaUserService: BTAUserService,
                                         cc: MessagesControllerComponents,
                                         confirmationView: confirmation
                                       )(implicit ec: ExecutionContext) extends FrontendController(cc) with I18nSupport with Logging {
 
-  def onPageLoad: Action[AnyContent] = (identify andThen getData andThen requireData).async { implicit request =>
+  def onPageLoad: Action[AnyContent] = (identify andThen getData andThen requireData).async{ implicit request =>
 
-    def show(c: Confirmation, pdf: PdfViewModel): Future[Result] = for {
+    def show(c: Confirmation, pdf: PdfViewModel): Future[Result] =
+      (for {
+      isBTAUser <- btaUserService.isBTAUser(request.internalId)
+      _ <- if(isBTAUser) btaUserService.remove(request.internalId) else Future.successful(false)
```

Comment:
this will make BTA users lose their back link if they go back to view their apps/rulings, might be worth checking with BA/design if this is supposed to happen

Reply:
fixed